tool/tsh/tsh.go

```diff
@@ -491,8 +492,9 @@ func onLogin(cf *CLIConf) {
 		if err != nil {
 			utils.FatalError(err)
 		}
+		key.TrustedCA = auth.AuthoritiesToTrustedCerts(authorities)
```

Comment:
Can you add a comment to this block of code about why we are doing this in-case we run into this again in the future.

Reply:
Added.
But that got me thinking - why do we only return current cluster CA in SSH login response?